tests/integration/telemetry/stats/prometheus/stats_filter_test.go

```diff
@@ -148,9 +142,11 @@ func setupConfig(cfg *istio.Config) {
 	if cfg == nil {
 		return
 	}
-	// disable telemetry and mixer filter
-	cfg.Values["global.disablePolicyChecks"] = "true"
-	cfg.Values["mixer.telemetry.enabled"] = "false"
+	// disable mixer telemetry and enable telemetry v2
+	cfg.Values["telemetry.enabled"] = "true"
+	cfg.Values["telemetry.v1.enabled"] = "false"
+	cfg.Values["telemetry.v2.enabled"] = "true"
+	cfg.Values["telemetry.v2.prometheus.enabled"] = "true"
```

Comment:
this is going to break in postsubmit I think

Reply:
Hmm which one? operator test or telemetry test? Why?

Reply:
we run these tests using helm in postsubmit and these don't exist in the helm install, so the stats filter will be missing

Reply:
I think the telemetry test uses operator: https://storage.googleapis.com/istio-prow/logs/integ-telemetry-k8s-tests_istio_postsubmit/1017/build-log.txt and https://github.com/istio/test-infra/blob/1cc5d5a20d8efcbe7adf6ae67c735b82cce42f2c/prow/cluster/jobs/istio/istio/istio.istio.master.gen.yaml#L1190. Am I looking at the wrong place?

Reply:
https://github.com/istio/test-infra/blob/802a1c55384dff4e5114f6f34ddc6572a9681e91/prow/config/jobs/istio.yaml#L244 is running them

Reply:
Ah I see, thanks! So should we add an `operator` label to annotate test suite that only runs with operator?

Reply:
I think instead of a label we could just have it check if --istio.kube.operator (or whatever the flag is) is set, then skip the test? But if that doesn't work label would as well

Reply:
is there a plan to move install charts to istio/istio? the two repo change is awkward to say the least.

Reply:
Yes, it should be done this week

Reply:
Those tests are actually passed :), because mixer is properly configured since`telemetry.v1.enabled` is missed in helm. https://github.com/istio/installer/blob/master/istio-control/istio-discovery/values.yaml#L155

Reply:
Ah great, that's how it actually should be, testing behavior not implementation. I assumed it was like all the other tests that don't do that :slightly_smiling_face: thanks!